### PR TITLE
[RPA-564] Add new SAP actions.

### PR DESCRIPTION
**Pull request by star-in-the-jar**

### CODE REVIEW

File: runbotics/runbotics-desktop/src/actions/sap/SAPAutomation.ts [hunk 1]
```diff
@@ -108,6 +112,25 @@ class SAPAutomation extends StatefulActionHandler {
         return {};
     }
 
+    async openContextMenu(
+        input: SapTypes.SAPOpenContextMenuActionInput,
+    ): Promise<SapTypes.SAPOpenContextMenuActionOutput> {
+        this.sessions['session'].FindById(input.target).pressContextButton(input.menuId);
+        return {};
+    }
```
Comment (preludium) [inline runbotics/runbotics-desktop/src/actions/sap/SAPAutomation.ts:120]: add empty lines between functions

File: runbotics/runbotics-orchestrator-ui/src/translations/pl/process.json [hunk 2]
```diff
@@ -406,6 +406,18 @@
...
     "Process.Details.Modeler.Actions.SAP.Select.Input": "Dane wejściowe",
     "Process.Details.Modeler.Actions.SAP.Select.Target": "Cel",
+    "Process.Details.Modeler.Actions.Sap.OpenContextMenu.Label": "SAP: Otwórz Menu Kontekstowe",
+    "Process.Details.Modeler.Actions.SAP.OpenContextMenu.Input": "Dane wejściowe",
+    "Process.Details.Modeler.Actions.SAP.OpenContextMenu.Target": "Cel",
+    "Process.Details.Modeler.Actions.SAP.OpenContextMenu.MenuId": "Id menu",
+    "Process.Details.Modeler.Actions.Sap.SelectFromContextMenu.Label": "SAP: Wybierz z Menu Kontekstowego",
+    "Process.Details.Modeler.Actions.SAP.SelectFromContextMenu.Input": "Dane wejściowe",
+    "Process.Details.Modeler.Actions.SAP.SelectFromContextMenu.Target": "Cel",
+    "Process.Details.Modeler.Actions.SAP.SelectFromContextMenu.OptionId": "Id opcji w menu",
```
Comment (preludium) [inline runbotics/runbotics-orchestrator-ui/src/translations/pl/process.json]: `Id opcji menu` should do the work

File: runbotics/runbotics-orchestrator-ui/src/utils/text.ts [hunk 3]
```diff
@@ -5,8 +5,15 @@ interface CapitalizeFirstLetterParams {
...
-        .split(delimiter)
-        .map((word) => word.charAt(0).toUpperCase() + (lowerCaseRest ? word.slice(1).toLowerCase() : word.slice(1)))
-        .join(join ?? '');
+export const capitalizeFirstLetter = ({
+    text,
+    lowerCaseRest = false,
+    delimiter,
+    join,
+}: CapitalizeFirstLetterParams) =>
+    !text
```
Comment (preludium) [inline runbotics/runbotics-orchestrator-ui/src/utils/text.ts]: let's go with `text.length === 0` + I believe that if there is possibility for any string then use `trim` also to this condition

**Review by preludium — DISMISSED** [review]

File: runbotics/runbotics-orchestrator-ui/src/translations/pl/process.json (hunk 2 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/translations/pl/process.json]: done

File: runbotics/runbotics-desktop/src/actions/sap/SAPAutomation.ts (hunk 1 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-desktop/src/actions/sap/SAPAutomation.ts:120]: done

File: runbotics/runbotics-orchestrator-ui/src/utils/text.ts (hunk 3 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/utils/text.ts]: done, I hope correctly

**Review by preludium — APPROVED** [review]

**Pull request merged**